packages/react-composites/src/composites/ChatComposite/adapter/AzureCommunicationFileUploadAdapter.ts

```diff
@@ -96,12 +98,12 @@ class FileUploadContext {
  */
 export class AzureCommunicationFileUploadAdapter implements FileUploadAdapter {
   private context: FileUploadContext;
-  private getAuthToken?: () => Promise<string>;
+  private credential?: CommunicationTokenCredential;
```

Comment:
Do we need the whole `CommunicationTokenCredential` here, rather than just the `getAuthToken` closure?